File: model/record.go
package model


type RecordForm struct {
	Offset		int	`json:"offset" form:"offset"`
	Number		int	`json:"number" form:"number"`
}


type Record struct {
	Label		string	`json:"label" form:"Label"`
	SubTitle	string	`json:"subtitle" form:"subtitle"`
	DateTime	string	`json:"datetime" form:"datetime"`
	Vid			string	`json:"vid" form:"vid"`
	Image		string	`json:"image" form:"image"`
}

File: page/src/model/record.ts
import BaseImple from './base'


export interface Record {
    label: string;
    subtitle: string;
    vid: string;
    datetime: string;
    image: string;
}

export interface RecordSetImple extends BaseImple {
    data: string;
}

export interface RecordGetImple extends BaseImple {
    data: Array<Record>;
}

export interface RecordReqModel {
    offset: number;
    number: number;
}

export interface RecordSaveModel {
    label: string;
    subtitle: string;
    image: string;
    text: string;
}
